bigquerydb/client.go

```diff
@@ -129,6 +136,7 @@ func (c *BigqueryClient) Write(timeseries []*prompb.TimeSeries) error {
 		ts := timeseries[i]
 		samples := ts.Samples
 		batch := make([]*Item, 0, len(samples))
+		c.recordsFetched.Add(float64(len(samples)))
```

Comment:
Do we really need a floating point number here? Would int64 make more sense?

Reply:
The go counter type is a float64 so we need to convert it.  https://godoc.org/github.com/prometheus/client_golang/prometheus#Counter